cpp/test/neighbors/ann_nn_descent.cuh

```diff
@@ -146,6 +189,125 @@ class AnnNNDescentTest : public ::testing::TestWithParam<AnnNNDescentInputs> {
   rmm::device_uvector<DataT> database;
 };
 
+template <typename DistanceT, typename DataT, typename IdxT>
+class AnnNNDescentBatchTest : public ::testing::TestWithParam<AnnNNDescentBatchInputs> {
```

Comment:
Do we have this consistently passing yet?

Reply:
No, that's a follow-on item. The tests are disabled for now both here and in RAFT.